### PR TITLE
Adds documentation and format files

### PR DESCRIPTION
**Pull request by r-akemii**
In this PR I would like to add a documentation module in docs for the processing providers you created.

I also propose renaming the module files to be more pythonic and removing the _icon.png_ file, once we can use the one in the qaequilibrae folder.

There are also proposals to modify some of the new translatable strings. The less new translatable strings we have, the best. Thus, I propose renaming the provider menu to match the one in qaequilibrae, and the tool names to be less verbose. What drove me to this set of modifications is thinking that today we're frequently updating the translations, but in the future, there can exist too many untranslated words for lack of maintenance.

### CODE REVIEW

**Comment by r-akemii** [comment]
@Art-Ev if you want to talk to me about any of these modifications, just let me know!

File: qaequilibrae/modules/processing_provider/Add_connectors.py [hunk 1]
```diff
@@ -1,65 +1,86 @@
...
-        return self.tr('Add connectors')
+        return self.tr("Add centroid connectors")
 
     def displayName(self):
-        return self.tr('Add connectors')
+        return self.tr("Add centroid connectors")
 
     def group(self):
-        return self.tr('1_Network')
+        return self.tr("Model Building")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/Add_connectors.py:77]: To keep menus in order we better use "1-Model Building"

File: qaequilibrae/modules/processing_provider/Add_connectors.py [hunk 2]
```diff
@@ -1,65 +1,86 @@
...
 
     def groupId(self):
-        return self.tr('1_Network')
+        return self.tr("Model Building")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/Add_connectors.py:80]: Proposal : "1-Model Building"

File: qaequilibrae/modules/processing_provider/assign_from_yaml.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        return {"Output": "Traffic assignment successfully completed"}
+
+    def name(self):
+        return self.tr("Traffic assignment from file")
+
+    def displayName(self):
+        return self.tr("Traffic assignment from file")
+
+    def group(self):
+        return self.tr("Paths and assignment")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/assign_from_yaml.py:133]: "3-Paths and assignment"

File: qaequilibrae/modules/processing_provider/assign_from_yaml.py [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+    def groupId(self):
+        return self.tr("Paths and assignment")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/assign_from_yaml.py:136]: "3-Paths and assignment"

File: qaequilibrae/modules/processing_provider/export_matrix.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        return {"Output": pathDest}
+
+    def name(self):
+        return self.tr("Export matrices")
+
+    def displayName(self):
+        return self.tr("Export matrices")
+
+    def group(self):
+        return self.tr("Data")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/export_matrix.py:72]: "2-Data"

File: qaequilibrae/modules/processing_provider/export_matrix.py [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+    def groupId(self):
+        return self.tr("Data")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/export_matrix.py:75]: "2-Data"

File: qaequilibrae/modules/processing_provider/matrix_from_layer.py [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+        return {"Output": output}
+
+    def name(self):
+        return self.tr("Import matrices")
+
+    def displayName(self):
+        return self.tr("Import matrices")
+
+    def group(self):
+        return self.tr("Data")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/matrix_from_layer.py:164]: "2-Data"

File: qaequilibrae/modules/processing_provider/matrix_from_layer.py [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+    def groupId(self):
+        return self.tr("Data")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/matrix_from_layer.py:167]: "2-Data"

File: qaequilibrae/modules/processing_provider/project_from_layers.py [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+        return {"Output": output_file}
+
+    def name(self):
+        return self.tr("Create project from link layer")
+
+    def displayName(self):
+        return self.tr("Create project from link layer")
+
+    def group(self):
+        return self.tr("Model Building")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/project_from_layers.py:186]: "1-Model Building"

File: qaequilibrae/modules/processing_provider/project_from_layers.py [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+
+    def groupId(self):
+        return self.tr("Model Building")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/project_from_layers.py:189]: "1-Model Building"

File: qaequilibrae/modules/processing_provider/renumber_from_centroids.py [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+        return {"Output": output_file}
+
+    def name(self):
+        return self.tr("Nodes from centroid")
+
+    def displayName(self):
+        return self.tr("Nodes from centroid")
+
+    def group(self):
+        return self.tr("Model Building")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/renumber_from_centroids.py:143]: "1-Model Building"

File: qaequilibrae/modules/processing_provider/renumber_from_centroids.py [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+
+    def groupId(self):
+        return self.tr("Model Building")
```
Comment (Art-Ev) [inline qaequilibrae/modules/processing_provider/renumber_from_centroids.py:146]: "1-Model Building"

**Review by Art-Ev — CHANGES_REQUESTED** [review]
Seems good to mee @r-akemii, juste some changes we need to make to keep control on menu order in qgis interface (proposals in review comments)

**Pull request closed without merging**